### ISSUE TITLE
Cannot find module package.json

### ISSUE THREAD

**Issue by rjferguson21** [defenseunicorns/pepr#16]
Installed pepr globally with `npm -g pepr@0.1.38`  and am seeing the following error:

```
[rob@lia pepr (main)]$ pepr init
node:internal/modules/cjs/loader:1056
  throw err;
  ^

Error: Cannot find module 'package.json'
Require stack:
- /opt/homebrew/lib/node_modules/pepr/dist/src/cli/build.js
- /opt/homebrew/lib/node_modules/pepr/dist/src/cli/index.js
- /opt/homebrew/lib/node_modules/pepr/dist/cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1053:15)
    at Module._load (node:internal/modules/cjs/loader:898:27)
    at Module.require (node:internal/modules/cjs/loader:1120:19)
    at require (node:internal/modules/helpers:112:18)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/pepr/dist/src/cli/build.js:10:24)
    at Module._compile (node:internal/modules/cjs/loader:1239:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1293:10)
    at Module.load (node:internal/modules/cjs/loader:1096:32)
    at Module._load (node:internal/modules/cjs/loader:935:12)
    at Module.require (node:internal/modules/cjs/loader:1120:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/homebrew/lib/node_modules/pepr/dist/src/cli/build.js',
    '/opt/homebrew/lib/node_modules/pepr/dist/src/cli/index.js',
    '/opt/homebrew/lib/node_modules/pepr/dist/cli.js'
  ]
}

Node.js v19.4.0
```

**Comment by jeff-mccoy** [comment]
Thanks, working on it right now--typescript doesn't deal with it's import aliases it seems.... 😭

**Comment by jeff-mccoy** [comment]
Okay this should be fixed in 0.1.41 now.

**Comment by rjferguson21** [comment]
Thanks - retested and looks like it is working now.
